### PR TITLE
Wire v10

### PR DESCRIPTION
**Pull request by mazzy89**
Signed-off-by: Salvatore Mazzarino <apocalipse89@gmail.com>

Wire v10

I put as azure-operator version 3.0.0 cause this will have k8s 1.15.

Feel free to correct it if I'm wrong

### CODE REVIEW

**Comment by MarcelMue** [comment]
Something is not right here - I can take over:
```
# github.com/giantswarm/azure-operator/service/controller
service/controller/cluster.go:216:6: v10ResourceSet declared and not used
```

**Comment by mazzy89** [comment]
I'll take a look asap

File: service/controller/v10/version_bundle.go [hunk 1]
```diff
@@ -61,6 +36,6 @@ func VersionBundle() versionbundle.Bundle {
 			},
 		},
 		Name:    "azure-operator",
-		Version: "2.5.0",
+		Version: "3.0.0",
```
Comment (MarcelMue) [inline service/controller/v10/version_bundle.go]:
```suggestion
		Version: "2.6.0",
```

Let's not bump too much here - we usually bump this high if there are many changes in the operator. This here is mainly in k8scc IMO.

Comment (mazzy89) [reply service/controller/v10/version_bundle.go]: ok it make sense

**Pull request merged**